uaa/slateCustomizations/source/index.html.md.erb

```diff
@@ -398,11 +398,18 @@ the UAA Introspect Token endpoint to validate that the tokens have not been revo
 (e.g. a few minutes), and when needed, revoke the more long-lived refresh tokens so that they
 may no longer be used to obtain refreshed access tokens.
 
-When a refresh token in the opaque format is successfully passed to the ``/revoke`` endpoint,
+When a refresh token is successfully passed to the ``/revoke`` endpoint,
 the refresh token can no longer be used to perform the Refresh Token grant.
-When a refresh token is in the JWT format, the server config ``uaa.jwt.revocable`` or
-the Identity Zone config ``config.tokenPolicy.jwtRevocable`` must be set to ``true`` for
-the "Revoke a single token" endpoint (``/oauth/token/revoke/{tokenId}``) to work.
+
+Refresh tokens in any format can be revoked using the "Revoke all tokens for a user" endpoint (``/oauth/token/revoke/user/{userId}``),
+the "Revoke all tokens for a client" endpoint (``/oauth/token/revoke/client/{clientId}``), or
+the "Revoke all tokens for a user and client combination" endpoint (``/oauth/token/revoke/user/{userId}/client/{clientId}``).
+
+Refresh tokens in the opaque format can be individually revoked using
+the "Revoke a single token" endpoint (``/oauth/token/revoke/{tokenId}``).
+However, refresh tokens in the JWT format can only be individually revoked using
```

Comment:
I forgot if you said you validated this one or not. Because this would be out of spec. As we validated yesterday, the spec says that refresh tokens MUST be revocable.

Reply:
This is validated (see story comment). This scenario would indeed be out of spec. However, UAA can be configured to be spec-compliant (when refresh token format is set of opaque). There are many other examples in UAA where it can be configured to be either spec compliant or out of spec (previous behavior, and hence no breaking change). So the fact that UAA can be configured to be out of spec is not an immediate problem in my mind. 

It is a problem that the default value of refresh token format is jwt (which would make UAA out of spec, when jwt.revocable = false, which is also the default value). But in any case, I see that as out of scope of the current doc improvement PR).

Reply:
issue filed: https://github.com/cloudfoundry/uaa-release/issues/813